src/components/picker/examples/picker.tsx

```diff
@@ -2,7 +2,9 @@ import { ListItem } from '@limetech/lime-elements';
 import { Component, h, State } from '@stencil/core';
 
 const NETWORK_DELAY = 500;
-
+/**
+ * Single value can be picked
+ */
```

Comment:
What about this one `Single value` ? The sentence doesn't look like a heading

Reply:
I think the new header explains the example more clearly.